tasks/main.yml

```diff
@@ -12,10 +12,11 @@
   apt: pkg=buildkite-agent state=present
 
 - name: Configure Buildkite
-  template: src=buildkite-agent.cfg.j2
-            dest="{{ buildkite_agent_conf_dir }}/buildkite-agent.cfg"
-            owner=buildkite-agent
-            group=buildkite-agent
-            mode=0600
+  template:
+    src: "buildkite-agent.cfg.v{{ buildkite_agent_major_version }}.j2"
```

Comment:
I initially had a problem with the previous syntax when I added the variable expansion to it - this version of the syntax makes it clearer, I think, to read and notice the need for quotes.